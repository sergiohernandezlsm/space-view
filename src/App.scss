.App {
  text-align: center;
  padding-top: 50px;
  color: #ffffff;
}

body {
  background-image: url(http://unblast.com/wp-content/uploads/2021/01/Space-Background-Image-2.jpg);
}
