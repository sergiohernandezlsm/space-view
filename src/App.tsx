import { Route, Redirect } from "react-router-dom";
import SearchPage from "./pages/search";
import AssetPage from "./pages/asset";
import { AssetsContextProvider } from "./store/assets-context";
import "./App.scss";

function App() {
  return (
    <AssetsContextProvider>
      <main className="App">
        <Route path="/" exact>
          <Redirect to="/search" />
        </Route>
        <Route path="/search" exact>
          <SearchPage />
        </Route>
        <Route path="/asset/:id" exact>
          <AssetPage />
        </Route>
      </main>
    </AssetsContextProvider>
  );
}

export default App;
