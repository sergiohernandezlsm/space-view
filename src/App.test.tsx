import { shallow } from "enzyme";
import App from "./App";

test("should render page", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
