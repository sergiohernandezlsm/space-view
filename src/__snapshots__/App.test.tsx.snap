// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page 1`] = `ShallowWrapper {}`;
